packages/grafana-ui/src/components/LoadingBar/LoadingBar.tsx

```diff
@@ -1,48 +1,45 @@
 import { css, keyframes } from '@emotion/css';
-import React from 'react';
-
-import { GrafanaTheme2 } from '@grafana/data';
+import React, { CSSProperties } from 'react';
 
 import { useStyles2 } from '../../themes';
 
 export interface LoadingBarProps {
-  width?: string;
-  height?: string;
```

Comment:
Is removing this height prop intended? Levitate reports a breakage in the API by removing it but I could not find any plugin using it at the moment.

I see that removing this prop removes some flexibility from the component and set a static height value. Could we leave this height customizable?

Reply:
@academo yes, but this is an internal component that not exposed anywhere

Reply:
at least I thought it was an internal implementation detail of PanelChrome. But looks like a standalone component, not sure why we did that given it's such an early state component that does not need to be exported initially. I think we should remove it from being exported until we finalize the design of it and see more use cases for it